qe/drivers.py

```diff
@@ -79,6 +79,34 @@ def integral_category(i, j, k, l):
         return "G"
 
 
+#  ___  _________ _____
+#  |  \/  || ___ \_   _|
+#  | .  . || |_/ / | |
+#  | |\/| ||  __/  | |
+#  | |  | || |    _| |_
+#  \_|  |_/\_|    \___/
+#
+
+
+def get_chunk(comm: MPI.COMM_WORLD, psi_global: Psi_det):
```

Comment:
`get_chunk_of_external_space` maybe more. 

In fact maybe  put this function inside the `gen_all_connected_determinant` ? Or at least near him